src/target/source/codegen_c_host.cc

```diff
@@ -20,10 +20,10 @@
 /*!
  * \file codegen_c_host.cc
  */
-#include <tvm/target/codegen.h>
+#include "codegen_c_host.h"
```

Comment:
Please put local headers on the bottom.

Suggested include sequence:
1. #include<tvm_header>
2. #include<std_header>
3. #include"local_header"

Reply:
done, though, can you explain why tvm_header and local_header are distinct things in this codebase? does tvm_header mean headers in `include`, since they are installed?

Reply:
I think tvm_headers are those header files in the include dir, and they are exposed to external developers; while local headers are those used internally, and they are designed to interface multiple .cc source code files. Just my humble opinion.